eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -71,11 +73,11 @@ jobs:
           targetPath: $(Build.SourcesDirectory)/eng/BuildConfiguration
           artifactName: BuildConfiguration
 
-    # Populate internal runtime variables.
-    - template: /eng/common/templates/steps/enable-internal-sources.yml
-      parameters:
-        legacyCredential: $(dn-bot-dnceng-artifact-feeds-rw)
-    - template: /eng/common/templates/steps/enable-internal-runtimes.yml
+    - ${{ if eq(parameters.populateInternalRuntimeVariables, true) }}:
```

Comment:
This also didn't need to run on every build. It only applies to internal builds. So, added the parameter to control that.